pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -319,6 +325,11 @@ func TestDesiredRouterDeployment(t *testing.T) {
 
 	checkDeploymentHasEnvVar(t, deployment, "ROUTER_USE_PROXY_PROTOCOL", true, "true")
 
+	checkDeploymentHasEnvVar(t, deployment, "ROUTER_UNIQUE_ID_HEADER_NAME", true, "unique-id")
```

Comment:
The env checks for `ROUTER_UNIQUE_ID_HEADER_NAME` and `ROUTER_UNIQUE_ID_FORMAT` were missing, and this PR just adds them independent of the header case changes, right?

Reply:
Right, I snuck that in (sorry!).

Reply:
No worries! Just wanted to make sure that was the case.